webapp/sources/rudder/rudder-web/src/main/elm/sources/Node.elm

```diff
@@ -25,11 +39,28 @@ update msg model =
     GetScoreDetails res ->
       case res of
         Ok scoreDetails ->
-          ( { model | scoreDetails = Just scoreDetails }
-          , Cmd.none
+          ( { model | details = scoreDetails }
+          , Cmd.batch (List.map (\d -> getDetails {name = d.name, details = d.details }) scoreDetails)
           )
         Err err ->
-          processApiError "Getting compliance score details" err model
+          processApiError "Getting score details" err model
+    ReceiveDetails name value ->
+        case Html.Parser.run value of
+          Ok htmlString ->
+            let
+               filterScript html =
+                   case html of
+                       Element "script" _ _ -> Text ""
+                       Element tag attributes children -> Element tag attributes (children |> List.filter filterScript)
+                       s-> s
```

Comment:
Is that a guard against potential script elements passed in the html string ? 
The library seems to disallows scripts (like Elm, it does not have it in the Html tree representation : https://stackoverflow.com/a/74528143), and maybe this check should be removed then